dist/src/main/resources/config/hawkular.properties

```diff
@@ -29,3 +29,4 @@ hawkular-alerts.engine-delay=1000
 hawkular-alerts.engine-period=2000
 hawkular-alerts.base-url=/hawkular/alerts
 #hawkular-alerts.backend-reindex=true
+#hawkular-alerts.prometheus-alerter=true
```

Comment:
Yes, I think we can set up enabled by default.
Also I think we can collect all configuration properties we have spreaded over several places and document them commented in the hawkular.properties file, now we have a nice one to do it.